### PR TITLE
MTA-1180: Release notes for v7.0.0

### PR DESCRIPTION
**Pull request by anarnold97**
### Jira

* [MTA-1180](https://issues.redhat.com/browse/MTA-1180)

### Preview

* [MTA 7.0.0 release notes](https://deploy-preview-804--windup-documentation.netlify.app/docs/release-notes/master/#rn-new-features-7-0-0_release-notes)

#### Reviews from earlier PR

* There are two languages at this time . Java and Go out of which Go is Tech Preview only.
* The new feature list is missing :

   1. Enhanced assessment module with custom questionnaire
   2. Grouping applications for assessment into archetypes.
   3. Unlink applications from JIRA

*

### CODE REVIEW

File: docs/topics/mta-rn-technical-changes-7-0-0.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes-7.0/master.adoc
+
+:_content-type: REFERENCE
+[id="mta-rn-technical-changes-7-0-0_{context}"]
```
Comment (anarnold97) [inline docs/topics/mta-rn-technical-changes-7-0-0.adoc:6]: [dylan](https://github.com/dymurray) && [shveta](https://github.com/sshveta)
Please can I ask for you input on any technical changes you think should be listed

File: docs/topics/mta-rn-resolved-issues-7-0-0.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+.Cannot create a Jira instance behind a proxy
+
+Cannot create a Red Hat Jira instance (issues.stage.redhat.com) through a proxy, but general proxy settings do not work with this configuration. link:https://issues.redhat.com/browse/MTA-849[MTA-849]
+
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12420807[Resolved Issues in Jira].
+
+////
+project in (MTA, WINDUP) AND status in (Verified, "Release Pending", Closed) AND priority in (Blocker, Critical, Major) AND fixVersion in ("MTA 7.0.0") AND component not in (documentation, QE-Task) ORDER BY description ASC
+////
```
Comment (anarnold97) [inline docs/topics/mta-rn-resolved-issues-7-0-0.adoc:23]:
```suggestion

```

**Comment by dymurray** [comment]
LGTM

**Review by abrugaro — APPROVED** [review]
LGTM

**Review by rromannissen — APPROVED** [review]
LGTM

File: docs/topics/mta-rn-known-issues-7-0-0.adoc [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-0_{context}"]
+= Known issues
+
+[ProductFullName] version 7.0.0 has the following issues.
+
+.Pathfinder assessment migration fails on upgrade from MTA 6.2.1 to MTA 7.0.0
```
Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-7-0-0.adoc:11]: @dymurray - included OK to publish

**Pull request merged**